tensorflow_datasets/video/moving_mnist.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import functools
+import numpy as np
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+_OUT_RESOLUTION = (64, 64)
+_SEQUENCE_LENGTH = 20
+_citation = """
+@article{DBLP:journals/corr/SrivastavaMS15,
+  author    = {Nitish Srivastava and
+               Elman Mansimov and
+               Ruslan Salakhutdinov},
+  title     = {Unsupervised Learning of Video Representations using LSTMs},
+  journal   = {CoRR},
+  volume    = {abs/1502.04681},
+  year      = {2015},
+  url       = {http://arxiv.org/abs/1502.04681},
+  archivePrefix = {arXiv},
+  eprint    = {1502.04681},
+  timestamp = {Mon, 13 Aug 2018 16:47:05 +0200},
+  biburl    = {https://dblp.org/rec/bib/journals/corr/SrivastavaMS15},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+
+
+class MovingMnist(tfds.core.GeneratorBasedBuilder):
+
+  VERSION = tfds.core.Version("0.1.0")
+
+  def _info(self):
+    return tfds.core.DatasetInfo(
+        builder=self,
+        description=(
+          "Moving variant of MNIST database of handwritten digits. This is the "
+          "data used by the authors for reporting model performance. See "
+          "`tfds.video.moving_sequence` for functions to generate training/"
+          "validation data."),
+        features=tfds.features.FeaturesDict(
+          dict(image_sequence=tfds.features.Video(
+              shape=(_SEQUENCE_LENGTH,) + _OUT_RESOLUTION + (1,)))),
+        urls=["http://www.cs.toronto.edu/~nitish/unsupervised_video/"],
+        citation=_citation,
+        splits=[tfds.Split.TEST]
```

Comment:
You shouldn't put the `splits=[tfds.Split.TEST]` as this will automatically be pre-computed from the `_split_generators` function.